tests/test_utils.py

```diff
@@ -457,7 +457,7 @@ def test_is_list_annotation(type_: Any, expected: Any) -> Any:
         ),
         pytest.param(
             DictConfig(ref_type=Any, content=ConcretePlugin),
-            Any,
+            Optional[Dict[Any, Any]],
             id="DictConfig[ConcretePlugin]_Any_reftype",
```

Comment:
Why did you change this?

Reply:
I didn't, I guess it was a fix in pre-commit.

Reply:
hmm, if you didn't change it it should not be in this PR.
make sure that you are rebased and that the PR contains only code that you changed.

Reply:
I didn't understand why you meant.
No,  pre-commit would not do that.
This is changing a test, that test is passing. It seems that this change is required due to a change in your logic.

Please figure out why this had to change.

Reply:
Oh no sorry, I forgot I changed it I changed _utils.get_ref_type so it supports the changes to Dictconfig and listconfig so now Dictconfig with Any ref_type would return Optional[Dict[Any, Any]] instead of Any.

Reply:
This change does not make sense to me:
Any means it can take any variable (including string or int).
Dict[Any, Any] means it can take any variable as the key and any variable in the value, but still must be an int.
This is actually not sensical for DictConfig because the key doesn't even support Any in general (it supports strings and enums).

Is Dict[Any, Any] a special sentinal value meaning Any? why not use Any then?

Reply:
Then should this be Any too ? `pytest.param(DictConfig(content={}), Optional[Dict[Any, Any]], id="DictConfig") `.

Reply:
This seems like it should be a Dict.
It also seems like it should be rejected because the dict key type does not support Any.